packages/Tabs/styles.jsx

```diff
@@ -0,0 +1,115 @@
+import styled, { css } from 'styled-components'
+
+export const TabsContainer = styled.div`
+  position: relative;
+
+  .react-tabs {
+    overflow-x: hidden;
+  }
+
+  .react-tabs__tab-list {
+    transition: 0.9s all ease;
+    transform: translate3d(${props => props.positionToMove}px, 0px, 0px);
+    white-space: nowrap;
+  }
+
+  .react-tabs__tab {
+    display: inline-block;
+    cursor: pointer;
+    padding: 0 25px;
+    min-width: 44px;
```

Comment:
We have specific pixel widths for padding/margins. Please see the section about spacing levels here: https://tds.telus.com/components/index.html#/Layout?id=box